modules/mono/editor/Godot.NET.Sdk/Godot.NET.Sdk/Sdk/Sdk.props

```diff
@@ -30,6 +30,15 @@
     <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
   </PropertyGroup>
 
+  <Target Condition=" $([MSBuild]::VersionLessThan($(MSBuildAssemblyVersion), '17.3')) " Name="GodotProjectDir" BeforeTargets="Build">
+    <Error Text="Cannot build from path containing '%23', move your project or update dotnet to the lastest version." Condition="$(GodotProjectDir.Contains('%23'))" /> <!-- # -->
```

Comment:
I'm not sure if you noticed the misspelling checks in the Files changed tab. They are reported by CI but don't cause CI to fail because they may cause false positives so we don't want to be too strict about it.

```suggestion
    <Error Text="Cannot build from path containing '%23', move your project or update dotnet to the latest version." Condition="$(GodotProjectDir.Contains('%23'))" /> <!-- # -->
```